src/script/ceph-release-notes

```diff
@@ -96,9 +110,14 @@ if __name__ == "__main__":
                         help="path to ceph git repo")
     parser.add_argument("--token", default=os.getenv("GITHUB_ACCESS_TOKEN"),
                         help="Github Access Token ($GITHUB_ACCESS_TOKEN otherwise)")
+    parser.add_argument("--original_issue", dest='original_issue', action='store_true',
```

Comment:
the only reason I didn't default that was when running against master branch etc. (ie non backport releases) this will significantly slow down things. But since the number of times this activity is done is relatively small I guess I can  change the defaults.